htdocs/langs/en_US/ticket.lang

```diff
@@ -137,6 +137,11 @@ TicketUseCaptchaCode=Use graphical code (CAPTCHA) when creating a ticket
 TicketUseCaptchaCodeHelp=Adds CAPTCHA verification when creating a new ticket.
 TicketsAllowClassificationModificationIfClosed=Allow to modify classification of closed tickets
 TicketsAllowClassificationModificationIfClosedHelp=Allow to modify classification (type, ticket group, severity) even if tickets are closed.
+TicketAutoCheckNotifyThirdParty=Notify third-party on ticket creation
```

Comment:
The description of this option seems wrong. Enablign the option does not send email. It seems it just auto-check a checkbox. So it just autoset a default value.